source/dagger/gameplay/ping_pong/player_scores.cpp

```diff
@@ -46,10 +46,17 @@ void PlayerScoresSystem::Run()
             ballOnField++;
         }
     }
-
     if (ballOnField == 0)
-    {
+    {   
+        
         // TODO: set speed to be random in both directions
-        CreatePingPongBall(s_TileSize, ColorRGBA(1, 1, 1, 1), { rand()%10 + 4,rand()%10 + 4,0 },   { 0,rand()%(s_FieldHeight / 2),0 });
+        if (k) {
```

Comment:
Is this meant to balance the game so that one ball goes left, and then the next goes right?

Reply:
yes

Reply:
Maybe go with `nextBallDirectionIsLeft` or something like that